install.ps1

```diff
@@ -387,7 +424,7 @@ function Add-ShimsDirToPath {
         }
 
         # For future sessions
-        [System.Environment]::SetEnvironmentVariable('PATH', "$SCOOP_SHIMS_DIR;$userEnvPath", 'User')
+        Set-Env 'PATH' "$SCOOP_SHIMS_DIR;$userEnvPath"
         # For current session
         $env:PATH = "$SCOOP_SHIMS_DIR;$env:PATH"
```

Comment:
Updating the Path will probably break and the current console session has to be restarted. Has this been tested?

Reply:
I only tried it on a virtual machine, and the `scoop` command can be used directly after installation through the `intall.ps1` script.

But the Powershell automated test script is too difficult for me, and I have no corresponding experience at all.

Reply:
> Updating `PATH` in the current session will probably break and the console has to be restarted. Has this been tested?

Sorry, which line are you asking about? I feel this PR shouldn't make any difference compared to before.

Reply:
I mean `$env:PATH = "$SCOOP_SHIMS_DIR;$env:PATH"`. We need to get the expanded string.
Or else it would result in `$env:PATH` being set to something like that:

![image](https://user-images.githubusercontent.com/432127/220938726-5f0518b8-8606-4e3c-aeb8-ae934c2b5535.png)

In this example I have just added an executable to the Downloads folder which **can't** be called after concatenating the users and system path without expanding the variables.

Reply:
How to test:
- Add an executable to downloads directory
- Add `%USERPROFILE%\Downloads` via the windows GUI
- Open powershell and run the command (I used `micro`)
- Copy and paste the `Get-Env` code from above
- Run `$env:PATH = "$(Get-Env 'PATH' -Global)$(Get-Env 'PATH')"` to update `$env:PATH`
- Try running the executable again, it will not run

Reply:
Scratch that... You're right it works. I missed it concatenates shims dir with `$env:PATH`... 😞

I was thinking it uses the values from the registry for that ... 😛